content/300-guides/050-database/100-developing-with-prisma-migrate/150-team-development.mdx

```diff
@@ -183,7 +183,12 @@ Your `schema.prisma` and local database now includes your team's changes, and th
 
 ## Source control
 
-If you are using Prisma Migrate, you must commit the following files to source control:
+You should commit the following files to source control:
 
 * The contents of the `.prisma/migrations` folder, including the `migration_lock.toml` file
 * The Prisma schema file (`schema.prisma`)
+
+Source-controlling the `schema.prisma` file is not enough - you must include your migration history. This is because:
+
+* As you start to [customize migrations](#customizing-migrations), your migration history contains **information that cannot be represented in the Prisma schema**. For example, you can customize a migration to mitigate data loss that would be caused by a breaking change.
+* The `prisma migrate deploy` command, which is used to deploy changes to staging, testing, and production environments, _only_ runs migration files. It does not use the Prisma schema file.
```

Comment:
>It does not use the Prisma schema file

Technically Migrate has to read it, because it needs to get the database provider and URL from there.